scipy/stats/_continuous_distns.py

```diff
@@ -1451,7 +1451,22 @@ def _shape_info(self):
 
     def _pdf(self, x):
         # cauchy.pdf(x) = 1 / (pi * (1 + x**2))
-        return 1.0/np.pi/(1.0+x*x)
+        with np.errstate(over='ignore'):
+            return 1.0/np.pi/(1.0+x*x)
+
+    def _logpdf(self, x):
+        # The formula
+        #     log(1/(pi*(1 + x**2))) = -log(pi) - log(1 + x**2)
+        #                            = -log(pi) - log(|x|*(1/|x| + |x|))
+        #                            = -log(pi) - (log(|x| + log(1/|x| + |x|)))
```

Comment:
Typo: one closing parentheses is misplaced

Reply:
Thanks. The comment has been rewritten in the latest update.